lib/Parser.js

```diff
@@ -299,6 +299,11 @@ function parseHeader(message, data, headerStart, headerEnd)
         message.event = parsed;
       }
       break;
+    // allow multiple call info headers
+    case 'call-info':
+      message.addHeader(headerName, headerValue);
```

Comment:
Why ' headerName'  instead of ' call-info' as in any other header above?

Reply:
According to RFC3261 (section 7.3.1, page 29 last paragraph on page 29)
not only the 'Call-Info' header can appear multiple times but any 
header which value can be expressed as a comma seperated list. Thus
the provided case block can be useable not only for the call-info but 
also for other headers which not need further special processing like 
in the other case blocks. Therefore I have used the `headerName` variable 
instead of the 'call-info' string literal. But can change this if you want.